cmd/crc/cmd/delete.go

```diff
@@ -15,8 +15,10 @@ import (
 var clearCache bool
 
 func init() {
-	deleteCmd.Flags().BoolVarP(&clearCache, "clear-cache", "", false,
-		fmt.Sprintf("Clear the OpenShift cluster cache at: %s", constants.MachineCacheDir))
+	if os.Getenv("CRC_PACKAGE") == "" {
```

Comment:
cache lives in `~/.crc/cache`, right? or is this also modified

Reply:
No it is changed https://github.com/code-ready/crc/pull/1827/files#diff-351617b3a8d07ea69e3c3430ca45061b17effc84aba09b482e8b8d79ee696a4eR110-R117 here